control/subscription_group.go

```diff
@@ -383,7 +377,7 @@ func (s *subscriptionGroup) subscribePlugins(id string,
 	for i, sub := range plugins {
 		plg, err := s.pluginManager.get(key(sub))
 		if err != nil {
-			serrs = append(serrs, serror.New(err))
+			serrs = append(serrs, pluginNotFoundError(sub))
```

Comment:
I'm looking through the code to see if there any chance the code could hit that error.